tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/core/Token.scala

```diff
@@ -32,27 +72,40 @@ case class Token(word: Symbol, properties: Map[Symbol, Set[Symbol]] = Map()) {
     * @return true if this token is considered punctuation
     */
   def isPunctuation: Boolean = {
-    getDeterministicProperty('cpos) == Symbol(".")
+    getDeterministicProperty(Token.coarsePos) == Symbol(".")
   }
 }
 
 object Token {
   implicit val tokenJsonFormat = jsonFormat2(Token.apply)
 
-  def createProperties(word: String, goldCpos: Option[String] = None): Map[Symbol, Set[Symbol]] = {
-    val propertyMap = Map('lcase -> Set(Symbol(word.toLowerCase)))
-    goldCpos match {
-      case Some(cpos) => propertyMap.updated('cpos, Set(Symbol(cpos)))
-      case None => propertyMap
-    }
-  }
+  def create(
+    word: String,
+    coarsePos: Option[String] = None,
+    finePos: Option[String] = None
+  ): Token = {
 
-  def create(word: String, coarsePos: String): Token = {
-    Token(Symbol(word), createProperties(word, Some(coarsePos)))
+    def createDefaultProperties(word: String, goldCpos: Option[String],
+      goldPos: Option[String]): Map[Symbol, Set[Symbol]] = {
+
+      val propertyMap = Map[Symbol, Set[Symbol]]()
+      val revisedMap = goldCpos match {
+        case Some(cpos) => propertyMap.updated(Token.coarsePos, Set(Symbol(cpos)))
```

Comment:
Why not just use `Map(Token.coarsePos, Set(Symbol(cpos))`? `propertyMap` is empty . . .